examples/run.sh

```diff
@@ -1,22 +1,22 @@
 #!/bin/bash -e
 
 echo "Running basic_file.py ..."
-python basic_file.py
+python examples/basic_file.py
```

Comment:
same here in the bash script. add the good old

```
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

and make use of that

Reply:
same as the previous comment but will take care of it.